docs/en/latest/concepts/apisix_route.md

```diff
@@ -21,18 +26,13 @@ title: ApisixRoute
 #
 -->
 
-`ApisixRoute` is a CRD resource which focus on how to route traffic to
-expected backend, it exposes many features supported by Apache APISIX.
-Compared to [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/),
-functions are implemented in a more native way, with stronger semantics.
+`ApisixRoute` is a Kubernetes CRD object that provides a spec to route traffic to services with APISIX. It is much more capable and easy to use compared to the default [Kubernetes Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) resource.
 
-Path based route rules
-----------------------
+See [reference](https://apisix.apache.org/docs/ingress-controller/references/apisix_route_v2/) for the full API documentation.
```

Comment:
I don't know why CI judged this link to be unavailable. do you know?  @juzhiyuan @bzp2010

Reply:
This is the error. See: https://github.com/apache/apisix-ingress-controller/actions/runs/3270950775/jobs/5381284169#step:3:117

```
ERROR: 1 dead links found!
[✖] file:///github/workspace/docs/en/latest/references/apisix_route_v2/.md → Status: 400
```

Reply:
Hi, @navendu-pottekkat. Maybe we need to remove the `/` at the end of the URL?

Reply:
It makes sense.